bitnami/mongodb/Chart.yaml

```
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mongodb
   - https://mongodb.org
-version: 13.5.0
+version: 13.5.1
```

Since it's a new feature addition, I would bump the minor version
```suggestion
version: 13.6.0
```